### PR TITLE
Standardize object declarations for 3d-tiles

### PR DESCRIPTION
**Pull request by lilleyse**
Continuation of #3338 for the 3d-tiles branch.

Also need to keep this in mind for the new classes in #3325.

In addition I removed two files that should have been removed from this branch earlier: `getBasePath.js` (turned into `getBaseUri.js` and `Scene/getMagic.js` (duplicate of `Core/getMagic.js`.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Part of #3177

File: Source/Scene/getMagic.js [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-        '../Core/getStringFromTypedArray'
-    ], function(
-        defaultValue,
-        getStringFromTypedArray) {
-    "use strict";
-
-    /**
-     * @private
-     */
-    var getMagic = function(uint8Array, byteOffset) {
```
Comment (pjcozzi) [inline Source/Scene/getMagic.js]: Aren't we still using this in `Model`?

Also, are we sure about removing `getBasePath`?

Comment (lilleyse) [inline Source/Scene/getMagic.js]: It was moved to `Source/Core/getMagic.js`

Also `getBasePath` was renamed to `getBaseUri`

Comment (pjcozzi) [inline Source/Scene/getMagic.js]: Ah, OK.  I missed the full path.

**Comment by pjcozzi** [comment]
Looks good.

**Pull request merged**